### PR TITLE
[KYUUBI #2043] Upgrade log4j/2.x/ to 2.17.2

### LINKED ISSUE #2043: [Improvement] Upgrade log4j/2.x/ to 2.17.2
### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### Search before asking

- [X] I have searched in the [issues](https://github.com/apache/incubator-kyuubi/issues?q=is%3Aissue) and found no similar issues.

### What would you like to be improved?

https://logging.apache.org/log4j/2.x/changes-report.html#a2.17.2

### How should we improve?

log4j.version from 2.17.1 to 2.17.2

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by RishiKumarRay**
### _Why are the changes needed?_

Close #2043

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [x] Add screenshots for manual tests if appropriate

  Screenshot from 2022-03-06 19-40-30

- [ ] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Review by yaooqinn — APPROVED** [review]
thanks for the contribution. LGTM

**Comment by cxzl25** [comment]
nit: title repeats a `#`

**Comment by yaooqinn** [comment]
> nit: title repeats a `#`

nice catch

**Review by ulysses-you — APPROVED** [review]

**Review by SteNicholas — APPROVED** [review]
LGTM. Please help to check the `SQLOperationListenerSuite` hanging.

**Comment by yaooqinn** [comment]
SQLOperationListenerSuite hangs

**Comment by SteNicholas** [comment]
@RishiKumarRay, could you please take a look at the CI for `SQLOperationListenerSuite` hanging.

**Comment by SteNicholas** [comment]
cc @turboFei

File: pom.xml [hunk 1]
```diff
@@ -133,7 +133,7 @@
         <scalatest.version>3.2.9</scalatest.version>
         <scopt.version>4.0.1</scopt.version>
         <slf4j.version>1.7.35</slf4j.version>
-        <log4j.version>2.17.1</log4j.version>
+        <log4j.version>2.17.2</log4j.version>
```
Comment (yaooqinn) [inline pom.xml:136]: hi @RishiKumarRay, you can add `<log4j.ignoreTCL>true</log4j.ignoreTCL>` in L1500 to fix this issue

Comment (yaooqinn) [reply pom.xml:136]: https://github.com/apache/spark/pull/35765 for more information

**Comment by RishiKumarRay** [comment]
@yaooqinn please check now

**Review by pan3793 — APPROVED** [review]

**Comment by yaooqinn** [comment]
@RishiKumarRay thanks for the first contribution.
I wish you can join the Kyuubi community for continuing contributions.

merged to master and 1.5

**Comment by RishiKumarRay** [comment]
@yaooqinn i can contribute with DevOps part as i have mostly worked with devops tools

**Comment by yaooqinn** [comment]
https://github.com/apache/incubator-kyuubi/pull/1918 maybe you can help some with the travis CI with arm64

**Comment by yaooqinn** [comment]
Also, we are also working on the admin UI, see the initial work https://github.com/apache/incubator-kyuubi/pull/2061 to check if you can help

**Comment by RishiKumarRay** [comment]
@yaooqinn yeah sure i will go through this , do you guys have discord?

**Comment by yaooqinn** [comment]
> @yaooqinn yeah sure i will go through this , do you guys have discord?

thanks for the help. we don't have discord, it's fully a community-driven project

**Comment by RishiKumarRay** [comment]
@yaooqinn you said something about joining community? how can i join ?

**Comment by yaooqinn** [comment]
actually, you are already in, there may be some stuff, like subscribe to a mailing list, that you can refer https://github.com/apache/incubator-kyuubi/blob/master/CONTRIBUTING.md.

We are still building community by this model https://github.com/apache/incubator-kyuubi/blob/master/MATURITY.md.

**Pull request closed without merging**